jobs/release.yml

```diff
@@ -3,12 +3,11 @@
 # ==================================================================================
 
 parameters:
-- name: SignTypeOverride
-  displayName: Signing Type Override
+- name: signType
+  displayName: Signing Type
   type: string
-  default: default
+  default: test
```

Comment:
Normally, I would think you would want to default to `real` signing, with `test` signing being a special case. We only test sign for internal-only builds, but I don't know if that matters. Up to you.